optuna_dashboard/ts/components/DataGrid.tsx

```diff
@@ -239,6 +187,76 @@ function DataGrid<T>(props: {
   )
 }
 
+function DataGridHeaderColumn<T>(props: {
+  column: DataGridColumn<T>
+  orderBy: Order | null
+  onOrderByChange: (direction: Order) => void
+  filtered: boolean
+  onFilterClear: () => void
+  dense?: boolean
+}) {
+  const { column, orderBy, onOrderByChange, filtered, onFilterClear, dense } =
+    props
+
+  const HiddenSpan = styled("span")({
+    border: 0,
+    clip: "rect(0 0 0 0)",
+    height: 1,
+    margin: -1,
+    overflow: "hidden",
+    padding: 0,
+    position: "absolute",
+    top: 20,
+    width: 1,
+  })
+  const TableHeaderCellSpan = styled("span")({
+    display: "inline-flex",
+  })
+  return (
+    <TableCell
+      padding={column.padding || "normal"}
+      sortDirection={orderBy || false}
+    >
+      <TableHeaderCellSpan>
+        {column.sortable ? (
+          <TableSortLabel
+            active={orderBy !== null}
+            direction={orderBy || "asc"}
+            onClick={() => {
+              if (orderBy === null) {
+                onOrderByChange("asc")
+              } else {
+                onOrderByChange(orderBy === "desc" ? "asc" : "desc")
+              }
```

Comment:
It could be simpler with this:

```suggestion
              onOrderByChange(orderBy === "asc" ? "desc" : "asc"
```